source/Octopus.Server.Client/Model/Triggers/FeedFilterResource.cs

```diff
@@ -9,4 +9,7 @@ public class FeedFilterResource : TriggerFilterResource
 
     [Writeable]
     public List<DeploymentActionPackageResource> Packages { get; set; } = new();
+
+    [Writeable]
+    public string FeedCategory { get; set; }
```

Comment:
Double checking - does this need to have a default value of empty string or will that get set on the other end if it's missing? (the server resource does have the default value)

Reply:
I don't think it needs it, but that's purely based off other resources that also do not set a default value.